<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSituationForm</class>
 <widget class="QFrame" name="CSituationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Situation</string>
  </property>
  <layout class="QVBoxLayout" name="vl_SituationForm">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_Situation">
     <property name="title">
      <string>Situation (PBH)</string>
     </property>
     <layout class="QVBoxLayout" name="vl_Situation">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QFrame" name="fr_Situation">
        <layout class="QGridLayout" name="gl_Situation" columnstretch="0,0,0,1">
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_Heading">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSlider" name="hs_Heading">
           <property name="minimum">
            <number>-179</number>
           </property>
           <property name="maximum">
            <number>180</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="3" alignment="Qt::AlignRight">
          <widget class="QPushButton" name="pb_SetOwnAircraft">
           <property name="text">
            <string>set</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="tb_ResetPitch">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../misc/misc.qrc">
             <normaloff>:/diagona/icons/diagona/icons/cross-white.png</normaloff>:/diagona/icons/diagona/icons/cross-white.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbl_GsKts">
           <property name="text">
            <string>Ground speed (kts):</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="sb_GsKts">
           <property name="suffix">
            <string>kts</string>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_Heading">
           <property name="text">
            <string>Heading (-179° to 180°):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="tb_ResetHeading">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../misc/misc.qrc">
             <normaloff>:/diagona/icons/diagona/icons/cross-white.png</normaloff>:/diagona/icons/diagona/icons/cross-white.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_Pitch">
           <property name="text">
            <string>0</string>
           </property>
           <property name="placeholderText">
            <string>deg.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_Bank">
           <property name="text">
            <string>0</string>
           </property>
           <property name="placeholderText">
            <string>deg.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSlider" name="hs_Pitch">
           <property name="minimum">
            <number>-179</number>
           </property>
           <property name="maximum">
            <number>180</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_Bank">
           <property name="text">
            <string>Bank (-179° to 180°):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="tb_ResetBank">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../misc/misc.qrc">
             <normaloff>:/diagona/icons/diagona/icons/cross-white.png</normaloff>:/diagona/icons/diagona/icons/cross-white.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_Pitch">
           <property name="text">
            <string>Pitch (-179° to 180°):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSlider" name="hs_Bank">
           <property name="minimum">
            <number>-179</number>
           </property>
           <property name="maximum">
            <number>180</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>25</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Coordinate">
     <property name="title">
      <string>Coordinate</string>
     </property>
     <layout class="QVBoxLayout" name="vl_Coordinate">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="swift::gui::editors::CCoordinateForm" name="comp_Coordinate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>175</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Environment">
     <property name="title">
      <string>Environment</string>
     </property>
     <layout class="QVBoxLayout" name="vl_Environment">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QFrame" name="fr_Environment">
        <layout class="QGridLayout" name="gl_Environment" columnstretch="0,0,0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_Pressure">
           <property name="text">
            <string>Pressure MSL (980 - 1046 mbar)</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_Pressure">
           <property name="text">
            <string>1013.00</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="tb_ResetPressure">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../misc/misc.qrc">
             <normaloff>:/diagona/icons/diagona/icons/cross-white.png</normaloff>:/diagona/icons/diagona/icons/cross-white.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSlider" name="hs_Pressure">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>980</number>
           </property>
           <property name="maximum">
            <number>1046</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1013</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3" alignment="Qt::AlignRight">
          <widget class="QPushButton" name="pb_SetEnvironment">
           <property name="text">
            <string>set</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="pb_PresetOwnAircraft">
     <property name="text">
      <string>reset to own aircraft values</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::editors::CCoordinateForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/coordinateform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Bank</tabstop>
  <tabstop>tb_ResetBank</tabstop>
  <tabstop>hs_Bank</tabstop>
  <tabstop>le_Pitch</tabstop>
  <tabstop>tb_ResetPitch</tabstop>
  <tabstop>hs_Pitch</tabstop>
  <tabstop>le_Heading</tabstop>
  <tabstop>tb_ResetHeading</tabstop>
  <tabstop>hs_Heading</tabstop>
  <tabstop>sb_GsKts</tabstop>
  <tabstop>pb_SetOwnAircraft</tabstop>
  <tabstop>le_Pressure</tabstop>
  <tabstop>tb_ResetPressure</tabstop>
  <tabstop>hs_Pressure</tabstop>
  <tabstop>pb_SetEnvironment</tabstop>
  <tabstop>pb_PresetOwnAircraft</tabstop>
 </tabstops>
 <resources>
  <include location="../../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
